src/router/index.js

```diff
@@ -1,9 +1,11 @@
 import Vue from 'vue';
 import VueRouter from 'vue-router';
-
+import SignUp from '../components/SignUp/index.vue';
```

Comment:
Into component folder, just should has one file index.vue, for the script and css file should other name. So here, you just write 'import SignUp from '../components/SignUp', mean it will find index file into folder SignUp to read

Reply:
render function not defined if do that